.github/workflows/terraformPlan.yml

```diff
@@ -0,0 +1,38 @@
+name: Ad-hoc Terraform Plan
```

Comment:
I'm not entirely sure I see the need for this workflow when you can just run the same locally. I wouldn't expect to run unit tests via a workflow.

Reply:
That's a great callout, and I absolutely understand where you are coming from. This was primarily created to allow for checking effects against different environments, since the `terraform_checks` additions only operate against PROD. Plus, it allows members who don't have direct Azure access to see the effects of code changes.

I'm perfectly fine removing this, though! This addition doesn't serve as much of a safety purpose as the other changes in this PR do.

Reply:
That's a really good point about being able to run `terraform plan` without Azure access. I'm fine with keeping this in that case.